packages/cloud-cognitive/src/components/DataSpreadsheet/getCellSize.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const getCellSize = (cellSize) => {
+  switch (cellSize) {
+    case 'compact':
```

Comment:
Not sure why we drift from the Carbon text box sizes of 32, 40, 48.

Either way, should we not use values exported by @carbon/layout (Which do not include 56).

I would certainly raise 56 as a non-standard size to design.

Note exports from @carbon/layout

```
var sizeXSmall = rem(24);
var sizeSmall = rem(32);
var sizeMedium = rem(40);
var sizeLarge = rem(48);
var sizeXLarge = rem(64);
var size2XLarge = rem(80);
```

Reply:
Good point about 56 not being a carbon default, I removed the xlg size for now